app/models/event.rb

```diff
@@ -0,0 +1,12 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  validates :description, presence: true, length: { minimum: 6 }
+  validates :date, presence: true
+  validates :title, presence: true
+
+  has_many :attendances, foreign_key: :attended_event_id, dependent: :destroy
+  has_many :attendees, through: :attendances
+
+  scope :past, -> { where('date<?', Time.now) }
+  scope :future, -> { where('date>?', Time.now) }
```

Comment:
This should be greater or equal to.  Kindly check below. ⬇️ 
`scope :upcoming, -> { where('date >= ?', Time.zone.now) }`